### ISSUE TITLE
Searches should prioritize records with accessibility metadata

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#39]
In the next version of our User Experience guide, I believe we should provide guidance on filtering for accessibility metadata. Those records which contain accessibility metadata should be displayed before others.

**Comment by HadrienGardeur** [comment]
I don't think that search ranking is necessarily the right approach as it's difficult to give a weight to such fields but a general recommandations regarding facets in search would be helpful.
